<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Institucion</class>
 <widget class="QDialog" name="Institucion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>406</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>406</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/prefijoNuevo/logo.png</normaloff>:/prefijoNuevo/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_3{
background-color: rgb(255, 255, 255);
border-Bottom-left-radius: 10px;
border-Bottom-right-radius: 10px;
}

QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 15px;
letter-spacing: 0.02em;
color: #6B7280;}


QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #3A4F50;
border-radius: 8px;
}


/* DATE */
QDateEdit
{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px;
}

QDateEdit::drop-down
{
	image: url(:/prefijoNuevo/025-calendar-13.png);
    width:15px;
    height:15px;
padding:13px 20px 10px 10px;
}

QDateEdit::focus{
background: #FFFFFF;
border: 2px solid #3A4F50;
border-radius: 8px;
}

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}

QComboBox {combobox-popup: 0}

QRadioButton{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
color:#9CA3AF}






</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="btn_guardar_institiucion">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>300</y>
           <width>120</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">/*  BOTON */

QPushButton{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 15px;
letter-spacing: 0.02em;
color:white;
padding:8px;
background: #3A4F50;
border-radius: 8px;}

QPushButton:hover{background: #344748;
border-radius: 8px; }
</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>21</height>
          </size>
         </property>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>461</width>
           <height>271</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>93</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Tipo de instituci贸n*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_nombre_institucion">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>36</y>
            <width>431</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>102</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese nombre de la instituci贸n</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>171</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Nombre instituci贸n*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_tipo_institucion">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>118</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese tipo de institucion</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>180</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Direcci贸n*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_direccion_institucion">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>204</y>
            <width>431</width>
            <height>40</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Ingrese la direcci贸n </string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl6_3">
          <property name="geometry">
           <rect>
            <x>242</x>
            <y>95</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Estado*</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
         <widget class="QComboBox" name="cbox_estado_intitucion">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>120</y>
            <width>205</width>
            <height>40</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Inactivo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Activo</string>
           </property>
          </item>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_2{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QToolButton" name="btn_closeedit">
         <property name="geometry">
          <rect>
           <x>445</x>
           <y>6</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Cerrar</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_closeedit{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeedit:hover{background:#ce4d3a ;
border-radius:11px; }

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../image/source.qrc">
           <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_titulo_ventana">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>4</y>
           <width>241</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
         </property>
         <property name="text">
          <string>Nueva Instituci贸n</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_instituto_seleccionado">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>27</y>
           <width>411</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_nombre_institucion</tabstop>
  <tabstop>line_tipo_institucion</tabstop>
  <tabstop>cbox_estado_intitucion</tabstop>
  <tabstop>line_direccion_institucion</tabstop>
 </tabstops>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
